thoth-export-server/src/bibtex/bibtex_thoth.rs

```diff
@@ -200,7 +201,14 @@ impl TryFrom<Work> for BibtexThothEntry {
                 .abstracts
                 .iter()
                 .find(|a| a.abstract_type == AbstractType::LONG)
-                .map(|a| a.content.clone()),
+                .map(|x| {
+                    convert_from_jats(
+                        &x.content,
+                        MarkupFormat::PlainText,
```

Comment:
Why not use `with_plain_text_abstracts` rather than getting JATS and then converting?